Firestore/core/src/core/target.h

```diff
@@ -38,6 +41,17 @@ class BundleSerializer;
 namespace core {
 
 using CollectionGroupId = std::shared_ptr<const std::string>;
+using IndexedValues = absl::optional<std::vector<google_firestore_v1_Value>>;
+
+struct IndexedBoundValue {
```

Comment:
Please add comments to these types. I also think IndexedBoundValue can be private (and should maybe just be `IndexBoundValue`)